pkg/virt-controller/watch/vmi.go

```diff
@@ -117,6 +118,8 @@ const (
 	// ImagePullBackOffReason is set when an error has occured while pulling an image for a containerDisk VM volume,
 	// and that kubelet is backing off before retrying.
 	ImagePullBackOffReason = "ImagePullBackOff"
+	// PVCUseByGuestfsReason is set when the PVC is used by a pod created by virtctl guestfs
+	PVCUseByGuestfsReason = "PVCUseByGuestfsReason"
```

Comment:
I wonder if this is too specific.  Should we have a new status such as 'Maintenance' to guard against all cases where a VM might be in use for some other purpose.  Think of tekton pipelines, VM import/export, etc.

Reply:
We still need to design the API and in the meantime, we need to prevent this situation. Additionally, this could cover the case for example when we want to perform disk operations before we create the VM.  Once we have the Maintenance mode we could put the VM in that state instead of throwing an error if we find a pod with these labels.

Instead of calling Guestfs I can use a more generic  name as Maintainance.